CONTRIBUTING.md

```diff
@@ -1,41 +1,75 @@
-# Contributing to `cosign`
+# Contributing to Cosign
 
-The following document will detail the steps needed in order to ensure all of the correct processes are followed when contributing to `cosign`.
+Thank you for considering contributing to Consign! We welcome any contributions, whether it's bug fixes, new features, or improvements to the existing codebase.
 
-The following are needed to be completed before a Pull Request is raised:
+## Your First Pull Request
 
-- unit tests written, run, and passing
-- lint run
-- document generation run
+Working on your first Pull Request? You can learn how from this free video series:
 
-## Building
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
 
-To build `cosign` locally, run the following:
+To help you get familiar with our contribution process, we have a list of [good first issues](https://github.com/sigstore/cosign/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) which are relatively limited scope. This is a great place to get started!
+
+Before working on an issue, please:
+
+1. Check the comments to see if someone is already working on it.
+1. If it's unassigned, comment that you're working on it to avoid duplication.
+
+## Contribution Prerequisites
+
+Before running Cosign, ensure that you have [Go](https://go.dev/doc/install) installed that includes `make`.
```

Comment:
Let's remove "that includes make", it can be assumed `make` is included in your OS distro.